sdk/core/Azure.Core/tests/Azure.Core.Tests.csproj

```diff
@@ -22,7 +22,6 @@
     <ProjectReference Include="..\src\Azure.Core.csproj" />
     <ProjectReference Include="..\..\Microsoft.Azure.Core.NewtonsoftJson\src\Microsoft.Azure.Core.NewtonsoftJson.csproj" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" />
-    <ProjectReference Include="..\..\..\textanalytics\Azure.AI.TextAnalytics\src\Azure.AI.TextAnalytics.csproj" />
```

Comment:
BTW, I added entry to .sln to fix this. Once mine merged, you can revert this one.

Reply:
just curious, what is the problem you are seeing with this reference?